keyboards/royal_kludge/rk61_rgb/info.json

```diff
@@ -0,0 +1,192 @@
+{
+    "keyboard_name": "RK61-RGB",
+    "manufacturer": "Royal Kludge",
+    "url": "http://en.rkgaming.com",
+    "maintainer": "fpb",
+    "bootloader": "sn32-dfu",
+    
+    "processor": "SN32F248BF",
+    "usb": {
+        "vid": "0x0C45",
+        "pid": "0x8016",
+        "device_version": "1.0.0",
+        "force_nkro": true
```

Comment:
```suggestion
```
forcing nkro will introduce issues in some machines. NKRO is negociated and enabled as needed. BIOS compatibility will probably be broken

Reply:
Done.